static/app/components/feedback/feedbackItem/screenshotSection.tsx

```diff
@@ -15,93 +14,79 @@ import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {Organization} from 'sentry/types';
 import type {Event} from 'sentry/types/event';
-import type {EventAttachment} from 'sentry/types/group';
-import {objectIsEmpty} from 'sentry/utils';
 
 type Props = {
   event: Event;
   organization: Organization;
   projectSlug: string;
 };
 
-export function ScreenshotSection({projectSlug, event, organization}: Props) {
+export function ScreenshotSection({event, organization, projectSlug}: Props) {
   const {screenshots} = useFeedbackScreenshot({projectSlug, event});
-  const hasContext = !objectIsEmpty(event.user ?? {}) || !objectIsEmpty(event.contexts);
   const {mutate: deleteAttachment} = useDeleteEventAttachmentOptimistic();
-  const [screenshotInFocus, setScreenshotInFocus] = useState<number>(0);
 
-  const handleDeleteScreenshot = useCallback(
-    (attachmentId: string) => {
-      deleteAttachment({
-        orgSlug: organization.slug,
-        projectSlug,
-        eventId: event.id,
-        attachmentId,
-      });
-    },
-    [deleteAttachment, event.id, organization.slug, projectSlug]
-  );
-
-  const handleOpenVisualizationModal = useCallback(
-    (eventAttachment: EventAttachment) => {
-      openModal(
-        modalProps => (
-          <OpenScreenshotModal
-            {...modalProps}
-            event={event}
-            orgSlug={organization.slug}
+  return screenshots.length ? (
+    <ScreenshotWrapper>
+      {screenshots.map(screenshot => (
+        <li key={screenshot.id}>
+          <FeedbackScreenshot
+            organization={organization}
             projectSlug={projectSlug}
-            eventAttachment={eventAttachment}
-            attachments={screenshots}
-            attachmentIndex={screenshotInFocus}
+            screenshot={screenshot}
+            onClick={() => {
+              openModal(
+                modalProps => (
+                  <OpenScreenshotModal
+                    {...modalProps}
+                    event={event}
+                    orgSlug={organization.slug}
+                    projectSlug={projectSlug}
+                    eventAttachment={screenshot}
+                    attachments={screenshots}
+                    attachmentIndex={screenshots.indexOf(screenshot)}
+                  />
+                ),
+                {modalCss}
+              );
+            }}
           />
-        ),
-        {modalCss}
-      );
-    },
-    [event, organization.slug, projectSlug, screenshotInFocus, screenshots]
-  );
-
-  if (!hasContext && !screenshots.length) {
```

Comment:
we don't need any of this `hasContext` stuff? :0

Reply:
i took it out because i think we really only care that there are screenshots. Within the rest of the page we don't show context at all, we get the message from `feedbackItem.metadata.message` for example. So I think if it's there or not, it shouldn't block our images. 

I'm a bit worried that we're assuming the attachments will be images though.

Reply:
could we validate somehow that they _are_ images?